markdown/usage/configuration.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Nextflow configuration
+subtitle: How configure Nextflow to work on your system
+---
+
+One of the strongest features of Nextflow is that it can run on virtually any computational infrastructure.
+It has built-in support for HPC execution schedulers such as Slurm, SGE, PBS, LSF and more as well as cloud compute infrastructure such as AWS Batch and Google Cloud.
+
+In order to get nf-core pipelines to run properly on your system, you will need to configure Nextflow so that it knows how best to run your analysis jobs.
+Each nf-core pipeline comes with "sensible defaults" for every configuration option. These are then overwritten as needed.
```

Comment:
May need to mention something about `executor` here or later i.e. if you have a cluster available you can create a config file that defines the relevant `executor`

Reply:
I agree - maybe also link to the Nextflow Docs on Executors from here then?

Reply:
Sounds good. We had quite a bit of detail on executors and I removed it. But would be good to mention it and link to the NF docs, agreed 👍